src/context.cpp

```diff
@@ -259,11 +259,11 @@ namespace Sass {
 
     // do not parse same resource twice
     // maybe raise an error in this case
-    if (sheets.count(inc.abs_path)) {
-      free(res.contents); free(res.srcmap);
-      throw std::runtime_error("duplicate resource registered");
-      return;
-    }
+    // if (sheets.count(inc.abs_path)) {
```

Comment:
> It might lead to memory leaks

This was my concern. It wasn't clear from the code that it was the case though.

> since importer could return different content for the same url this way

This is feature people rely on in custom importers so we need to support it in some fashion.